lms/djangoapps/django_comment_client/management/commands/rename_user.py

```diff
@@ -0,0 +1,22 @@
+"""
+Change the username of an existing user
+"""
+
+from django.core.management.base import BaseCommand, CommandError
+from .. import management_utils
+
+
+class Command(BaseCommand):
+    """
+    invoke this manage.py command from the console as follows:
+    python manage.py lms rename_user <username1> <username2> --settings=devstack
```

Comment:
While the `--settings=devstack` works for local testing, this will be
misleading when attempting to run it in a different environment. We
probably want to remove this bit.